### DISCUSSION TITLE
Welcome to ScrapMechanicCraftingCalculator Discussions!

### DISCUSSION THREAD

**Discussion by GandalftheGreyt3719** [GandalftheGreyt3719/ScrapMechanicCraftingCalculator#2] [Announcements]
* Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and be open-minded. 

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by HumanTheHuman** [comment]
I AM HUMANTHEHUMAN. ☻☻☻☻
